sdk/lusid_drive/utilities/file_streaming.py

```diff
@@ -0,0 +1,47 @@
+import re
+import six
+import requests
+from lusid_drive.utilities.lusid_drive_retry import lusid_drive_retry
+
+
+@lusid_drive_retry
+def stream_file_upload(api_factory, x_lusid_drive_filename, x_lusid_drive_path, content_length, body):
+    headers = {
+        "Accept": "application/json, text/plain, text/json",
+        "X-LUSID-SDK-Language": "Python",
+        # extracts a version from generated header
+        "X-LUSID-SDK-Version": api_factory.api_client.default_headers["User-Agent"][18:-7],
```

Comment:
use `.get()` rather than dict accessor in case the header values aren't there and only add the headers if they are